snippets/snippets.json

```diff
@@ -42,5 +42,10 @@
 		"body": [
 			"const ${1:memoizedValue} = useMemo(() => ${2:computeExpensiveValue}(${3:a}, ${4:b}), [${5:a}, ${6:b}])$0"
 		]
-	}
-}
\ No newline at end of file
+	},
+	"Import React, { useState,useContext,useEffect }": {
+    "prefix": "imrus",
```

Comment:
Thanks for the pull request! I like this, just curious why you picked `imrus` for the prefix?

Reply:
Import react and hooks , you can change it as you wanted.